pkg/util/domainutil/repair_vars.go

```diff
@@ -53,6 +54,30 @@ func (r *repairInfo) GetRepairTableList() []string {
 	return r.repairTableList
 }
 
+// GetMustLoadRepairTableListByDB gets must load repair table ID list.
+func (r *repairInfo) GetMustLoadRepairTableListByDB(dbName string, tableName2ID map[string]int64) []int64 {
+	r.RLock()
+	defer r.RUnlock()
+	dbNamePrefix := dbName + "."
+	repairTableSet := make(map[string]struct{}, len(r.repairTableList))
+	for _, fullTableName := range r.repairTableList {
+		lowerFullTableName := strings.ToLower(fullTableName)
+		if strings.HasPrefix(lowerFullTableName, dbNamePrefix) {
+			repairTableSet[lowerFullTableName] = struct{}{}
+		}
+	}
+
+	var tableIDList []int64
+	// tableName2ID is case sensitive and needs to be traversed to match the table id
+	for tableName, id := range tableName2ID {
+		fullName := fmt.Sprintf("%s.%s", dbName, tableName)
```

Comment:
For such simple string processing, you can just use the plus sign to concatenate strings.

https://forum.golangbridge.org/t/fmt-sprintf-vs-string-concatenation/23006